pkg/server/server_presentation_test.go

```diff
@@ -323,14 +323,14 @@ func TestPresentationAPI(t *testing.T) {
 				{
 					Status: "pending",
 					PresentationSubmission: &exchange.PresentationSubmission{
-						ID:           operation.SubmissionID(op.ID),
+						ID:           submission.ResourceID(op.ID),
 						DefinitionID: definition.PresentationDefinition.ID,
 					},
 				},
 				{
 					Status: "pending",
 					PresentationSubmission: &exchange.PresentationSubmission{
-						ID:           operation.SubmissionID(op2.ID),
+						ID:           submission.ResourceID(op2.ID),
```

Comment:
why resource over submission? submission is more meaningful in context

Reply:
Renamed to `ID`, as submission is already in the package name.